src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy", Locale.ENGLISH);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        double[] salary = new double[names.length];
+        LocalDate dateFromLD = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateToLD = LocalDate.parse(dateTo, formatter);
+        StringBuilder sb = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder report = new StringBuilder();
```
try to name variables according to their purpose